operators/index.js

```diff
@@ -64,7 +64,7 @@ function author(value, opts = { dedicated: false }) {
       prefix: 32,
       prefixOffset: AUTHOR_PREFIX_OFFSET,
       indexType: 'value_author',
-      version: 2,
+      version: 3,
```

Comment:
@rabble Not this one. I had mentioned specifically line 20 on indexes/about-self.js https://github.com/ssb-ngi-pointer/ssb-db2/pull/316/files#r827745805 Could you revert this one? Otherwise we would cause the value_author index to be rebuilt in vain.